docs/reference/index.md

```diff
@@ -2,6 +2,18 @@
 
 ```{toctree}
 :maxdepth: 1
+:hidden:
 
-jubilant
+jubilant module <jubilant>
+statustypes module <statustypes>
```

Comment:
Leaving a comment here for lack of a better place: I don't know how to get the types in the [jubilant.Status](https://canonical-jubilant--61.com.readthedocs.build/reference/jubilant/#jubilant.Status) signature to be linked to the [statustypes reference](https://canonical-jubilant--61.com.readthedocs.build/reference/statustypes/). Any ideas?

Reply:
Dima found [this Sphinx issue](https://github.com/sphinx-doc/sphinx/issues/12695), which looks like it's the exact issue. Unfortunately there's not a good workaround yet short of excluding that method/class from autodoc and writing the signature manually, which I don't want to do for these. I'll just leave it as is for now.

Reply:
This is what `description` looks like, FYI:

![image](https://github.com/user-attachments/assets/8cc2087b-6563-46c0-ab0e-2f1917afa0c6)

It takes considerably more space, although perhaps not quite so much when compared with the wrapping that gets done except for dataclasses.

Reply:
Ah, thanks. I've tried it locally, and while it's better for this, it also means all the dataclass parameters are repeated. There's probably a way to address that. But I think the cons outweigh the pros for now ... we can always revisit later.